packages/grid/x-data-grid-pro/src/useDataGridProComponent.tsx

```diff
@@ -74,6 +90,16 @@ export const useDataGridProComponent = (
   /**
    * Register all state initializers here.
    */
+  useGridInitializeState(columnMenuStateInitializer, apiRef, props);
```

Comment:
Could we keep the same order as the feature hook themselves ?
In 90% of the case, the order don't matter here.
But if one day we want to launch plugins and therefore loop on a list of objects to call those `useGridInitializeState` then the main hook, it will be easier if the order is always the same.